community/pike/APKBUILD

```diff
@@ -0,0 +1,200 @@
+# Maintainer: Paweł Tomak <pawel@tomak.eu>
+pkgname=pike
+pkgver=8.0.610
+pkgrel=0
+pkgdesc="Pike Programing language"
+url="https://pike.lysator.liu.se"
+arch="x86_64"
+license="GPL-2.0-or-later"
+options="!check" # there is no tests for those packages
+depends=
+makedepends="gmp-dev \
+			bzip2-dev \
+			freeglut-dev \
+			gdbm-dev \
+			giflib-dev \
+			giflib-utils \
+			gtk+2.0 \
+			libgcrypt-dev \
+			libglade-dev \
+			libsm-dev \
+			mesa-gl \
+			nettle \
+			nettle-dev \
+			nettle-utils \
+			pcre-dev \
+			sdl-dev \
+			sdl_mixer-dev \
+			sqlite-dev \
+			sqlite-libs \
+			readline \
+			zlib-dev \
+			librsvg-dev"
```

Comment:
It'd not needed to escape newlines inside a string. Please remove these backslahes and use only one tab for indentation.